frontend/src/pages/ApplicationForm/ApplicationForm.jsx

```diff
@@ -383,21 +383,17 @@ const ApplicationForm = props => {
                 <Grid.Column width={6}>
                   <Segment>
                     <SubTitle>Important Information</SubTitle>
-                    <List ordered>
+                    <List>
                       <ListItem>
-                        Our application includes a $10 printing fee to be submitted when you pick up your materials.*
+                       Application is on a rolling basis.  Once your application is submitted, you will be notified of your application status within 10 business days. 
```

Comment:
Hi @kailanjian - a small nit: are 2 spaces here intentional? I noticed 1 space after the `.` in the [following](https://github.com/International-Mentorship-ISMP/ismp/pull/213/files#diff-7015af09301fb9f2922aa34adff436edR391).
```suggestion
                       Application is on a rolling basis. Once your application is submitted, you will be notified of your application status within 10 business days. 
```

Reply:
it was not intentional, I just copied from basecamp 😂